ocfweb/stats/accounts.py

```diff
@@ -66,63 +80,8 @@ def _get_account_stats():
         cumulative_group_accounts.append(group_total)
 
         cur += one_day
-
+    dates = [time.mktime(d.timetuple())*1000 for d in dates]
```

Comment:
where does the 1000 come from? Is there a better way to implement this, such as inside the loop where `dates` is populated to begin with?

Reply:
Highcharts wants time in milliseconds. It's inside the loop now.